src/main/java/at/hannibal2/skyhanni/data/SackAPI.kt

```diff
@@ -328,6 +329,14 @@ object SackAPI {
         fetchSackItem(this).takeIf { it.statusIsCorrectOrAlright() }?.amount
 
     fun NEUInternalName.getAmountInSacks(): Int = getAmountInSacksOrNull() ?: 0
+
+    fun testSackAPI (args: Array<String>) {
+        if (args.size == 1 ) {
+            val amountInSacks = args[0].asInternalName().getAmountInSacksOrNull()
+            ChatUtils.chat("Amount of ${args[0]} in sacks: $amountInSacks")
```

Comment:
Instead can we just print the whole sack item data class so there is also info on status and last change.